src/app/cli/src/graphql.ml

```diff
@@ -552,67 +591,94 @@ module Make (Commands : Coda_commands.Intf) = struct
           let%map pk = Program.wallets coda |> Secrets.Wallets.generate_new in
           Result.return pk )
 
+    let user_command coda {Account.Poly.nonce; _} sender_kp memo payment_body
+        fee =
+      let payload =
+        User_command.Payload.create ~fee ~nonce ~memo ~body:payment_body
+      in
+      let payment = User_command.sign sender_kp payload in
+      let command = User_command.forget_check payment (*uhhh*) in
```

Comment:
uhh?

Reply:
lol I think that was me thinking that it seemed super weird to sign and then "forget you signed" but not knowing how else to do it